js/google_ima.js

```diff
@@ -416,13 +416,15 @@ require("../html5-common/js/utils/utils.js");
                   "adManager": this.name,
                   "ad": ad,
                   "streams": streams,
-                  "adType": _amc.ADTYPE.UNKNOWN_AD_REQUEST
+                  "adType": _amc.ADTYPE.UNKNOWN_AD_REQUEST,
+                  "mainContentDuration": this.mainContentDuration
                 };
 
                 //percentage position types require a different calculation.
```

Comment:
Would probably just update this comment to say that this is handled by the AMC, looks good otherwise.

Reply:
Removed. since it no longer applies.